extensions/amp-story/1.0/animation.js

```diff
@@ -74,52 +76,66 @@ const PlaybackActivity = {
   FINISH: 1,
 };
 
+/**
+ * @param {!Element} root
+ * @param {!Element} element
+ * @return {?string}
+ */
+function getSequencingStartAfterId(root, element) {
```

Comment:
Is there any reason why this is a standalone function rather than an AnimationManager method?

Reply:
It's pure, so it doesn't need to be a method. It also minifies better.